module/actor-sheet.js

```diff
@@ -337,19 +337,19 @@ export class CypherActorSheet extends ActorSheet {
         let brackets = "";
         // let description = "";
         let description = "<hr style='margin:3px 0;'><img class='description-image-chat' src='" + item.img + "' width='50' height='50'/>" + item.data.description;
-        let points = " points";
+        let points = " " + game.i18n.localize("CYPHERSYSTEM.Points");
         let notes = "";
         if (item.data.notes != "") notes = ", " + item.data.notes;
         if (item.type == "skill" || item.type == "teen Skill") {
           brackets = " (" + item.data.skillLevel + ")";
         } else if (item.type == "power Shift") {
-          brackets = " (" + item.data.powerShiftValue + " Shifts)"
+          brackets = " (" + item.data.powerShiftValue + " " + game.i18n.localize("CYPHERSYSTEM.Shifts") + ")"
         } else if (item.type == "ability" || item.type == "teen Ability") {
-          if (item.data.costPoints == "1") points = " point"
+          if (item.data.costPoints == "1") points = " " + game.i18n.localize("CYPHERSYSTEM.Point")
           if (item.data.costPoints != 0 && item.data.costPoints != 0) brackets = " (" + item.data.costPoints + " " + item.data.costPool + points + ")"
         } else if (item.type == "attack") {
-          if (item.data.damage == 1) points = " point"
-          let damage = ", " + item.data.damage + points + " of damage"
+          if (item.data.damage == 1) points = " " + game.i18n.localize("CYPHERSYSTEM.Point")
+          let damage = ", " + item.data.damage + points + " " + game.i18n.localize("CYPHERSYSTEM.OfDamage")
```

Comment:
See other sheet. I should really re-factor the sheets...

Reply:
Already answered, will change it

Reply:
Refactored translation